lib/chewy/index/observe/mongoid_methods.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Chewy
+  class Index
+    module Observe
+      extend Helpers
+      module MongoidMethods
+        class_methods do
+          def update_index(type_name, *args, &block)
```

Comment:
@mightymercado do we use this method anywhere?
This was removed from the adapter in the new version

Reply:
I don't think we do. You should be able to remove it if it was removed from chewy.

Reply:
Actually the monogid adapter itself was removed from chewy and we had to add it back for our usecase